src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -537,6 +538,9 @@ private function createAuthenticationListeners($container, $id, $firewall, &$aut
                     } elseif ('remember_me' === $key) {
```

Comment:
Careful when merging up: this condition also includes `anonymous === $key` as of 4.4, it needs to be split so that this applies only to remember_me.